apps/meteor/app/slackbridge/server/SlackAPI.js

```diff
@@ -32,8 +34,10 @@ export class SlackAPI {
 	async getGroups(cursor = null) {
 		let groups = [];
 		const request = await fetch('https://slack.com/api/conversations.list', {
+			headers: {
+				Authorization: `Bearer ${this.token || 'token'}`,
```

Comment:
Uhh, what's this ``token'` string?

Reply:
Ops! :eyes: It slipped through my eyes. Fixed it.